README.md

```diff
@@ -61,7 +61,7 @@ _Add your project ideas below:_
   and plan to get involved with the kubernetes community in fixing
   them.
 - Recommended Skills: Go, Kubernetes
-- Mentor(s): Justin Santa Barbara (@justinsb), Leigh Capili (@stealthybox)
+- Mentor(s): Justin Santa Barbara (@justinsb), Leigh Capili (@stealthybox), Sandeep Rajan (@rajansandeep)
```

Comment:
One of the subproject maintainers (eg. @justinsb) has to approve this.

Reply:
I discussed this with @stealthybox, who is one of the maintainers and mentor for this project

Reply:
Great, so please let @stealthybox approve this.

Reply:
+1